web-components/src/components/organisms/ProjectFilters/ProjectFilter.TagFilter.tsx

```diff
@@ -0,0 +1,40 @@
+import { cloneElement } from 'react';
+
+import { ProjectTag } from '../../molecules/ProjectTag/ProjectTag';
+
+interface TagFilterProps {
+  icon: JSX.Element;
+  name: string;
+  isSelected: boolean;
+  onClick?: () => void;
+}
+
+const tagSx = {
+  cursor: 'pointer',
+  boxShadow:
+    // eslint-disable-next-line lingui/no-unlocalized-strings
```

Comment:
Instead of using an inline disable, we could add the `boxShadow` property to the ignore ist.